app/controllers/cards_controller.rb

```diff
@@ -35,6 +45,10 @@ def create
 
   def update
     @card = Card.find_by(id: params[:id])
+    @card.board = Board.find_by(name: params[:board_name])
+    if @card.board.nil?
+      @card.board = Board.create_new_board params[:board_name]
```

Comment:
I'm not sure that we need to implicitly create Boards as part of the update action.If we're updating a `Card` it must already exist, and I'm not sure how it could without being attached to a `Board`